docs/packages/connect/methods/getDeviceState.md

```diff
@@ -19,8 +19,8 @@ Get the state of a device.
 ```javascript
 const capturedDevice = await TrezorConnect.getDeviceState({
```

Comment:
please split this to at least 2 commits. we use git log to generate connect changelog and we don't want to see there stuff like `feat(suite)` 🙏

Reply:
done

Reply:
Should all connect code be in separate commits, @mroz22?

Reply:
Good point, I have split the all `connect` code.